projects/experimental/pipes/amount/amount.pipe.ts

```diff
@@ -0,0 +1,45 @@
+import {Inject, Pipe, PipeTransform} from '@angular/core';
+import {tuiFormatCurrency, tuiFormatSignSymbol} from '@taiga-ui/addon-commerce';
+import {
+    TUI_NUMBER_FORMAT,
+    tuiFormatNumber,
+    TuiNumberFormatSettings,
+} from '@taiga-ui/core';
+
+import {TUI_AMOUNT_OPTIONS, TuiAmountOptions} from './amount.options';
+
+@Pipe({
+    name: `tuiAmount`,
+})
+export class TuiAmountPipePipe implements PipeTransform {
+    constructor(
+        @Inject(TUI_AMOUNT_OPTIONS) private readonly options: TuiAmountOptions,
+        @Inject(TUI_NUMBER_FORMAT) private readonly format: TuiNumberFormatSettings,
+    ) {}
+
+    transform(
+        value: number,
+        currency = this.options.currency,
+        currencyAlign = this.options.currencyAlign,
+    ): string {
+        const sign = tuiFormatSignSymbol(value, this.options.sign);
+        const currencySymbol = tuiFormatCurrency(currency);
+        const formatted = tuiFormatNumber(Math.abs(value), {
+            ...this.format,
+            decimalLimit: this.getDecimalLimit(value),
+        });
+        // eslint-disable-next-line no-irregular-whitespace
```

Comment:
It should be non breaking space, otherwise it could wrap to the next line.

Reply:
Yes, this exception is just for use non breaking space. Linter doesn't allow nbsp. Maybe disable this rule?

Reply:
Don't we have a const for it?